tools/js-sdk-release-tools/src/test/apiVersion/testCases/rlc-source-from-readme/src1/parameters.ts

```diff
@@ -0,0 +1,4 @@
+export interface XQueryParamProperties {
+    /** Api Version */
+    "api-version": "9999-01-01-preview";
```

Comment:
not sure we need this but actually api-version in parameters.ts file could be multiple.

Reply:
this is declaration file, I don't use this one. 
I use this one: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/compute/arm-compute-rest/src/parameters.ts#L113